<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>81</width>
     <height>211</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Настройки лаунчера и игры Minecraft</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ram_select">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>231</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1024</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2048</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4096</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6142</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>RAM : </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>MB</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Тема :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="theme">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>231</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Тёмная тема</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Яркая тема</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Язык :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="lang">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>231</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>RUru</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ENus</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Расположение :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="dir">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>200</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>~/Documents/north-launcher/instances</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>~/Документы/north-launcher/instances</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>~/.minecraft/nlauncher/instances</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>~/home/nlauncher/instances</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
